Logisk fungerande brukargrensesnitt, men knappefunksjonar må testast.

espi3000 opened the pull request:
Brukargrensesnitt er så godt som ferdig og testa på Arduino med vanlige I/O-funksjonar.

@e-sivertsen Det ligger ei ny fil under Debugging i denne branchen (gui) som må testast. Fila er som main, berre at alt unødvendig er tatt bort. NB! Hugs "no line ending" i seriellmonitor.

Ønskjer tilbakemelding om:

Det i det heile tatt fungerer
Det er noko som kan forbetre brukarvennligheten. t.d. delay.

espi3000 committed:
Logisk fungerande brukargrensesnitt, men knappefunksjonar må testast.

e-sivertsen commented:
@espi3000 

https://photos.app.goo.gl/dxFgAGnmHzDzDjXK6

Tydelig men varierende grad av _interlacing&#8482;_.

Knappetrykk fungerer bra.

Ser ut som om den setter seg fast i calibrate funksjonen?

espi3000 commented:
> @espi3000
> 
> https://photos.app.goo.gl/dxFgAGnmHzDzDjXK6
> 
> Tydelig men varierende grad av _interlacing™_.
> 
> Knappetrykk fungerer bra.
> 
> Ser ut som om den setter seg fast i calibrate funksjonen?

Ser ut som eg har gløymd nokon ting her ja. Skal prøve å fikse dette i løpet av helga.

espi3000 committed:
Update zumo_brukargrensesnitt.ino

Har prøvd å fikse interlacing og at kalibrering ikkje stopper av seg sjølv, men ikkje at berre deler av orda blir vist på displayet. Eg har ein funksjon i tankane som autoscroller orda på same måte som eit "dab-radio-display". Eventuelt kan me lage forkortelsar som passer på displayet.

B - pauser programmet
A - konfigurerer
C - konfigurerer

Programmet vil starte med konfigurasjon i seriellmonitoren.
Programmet skal gå rett i konfigurasjon etter kalibrering.

@e-sivertsen

espi3000 committed:
Update zumo_brukargrensesnitt.ino

Prøver igjen. @e-sivertsen

espi3000 committed:
Foreløpig ferdig brukargrensesnitt

espi3000 committed:
Foreløpig ferdig brukargrensesnitt

espi3000 committed:
Merge branch 'gui' of https://github.com/IELET1002-G2/The-Tesla-of-Zumo-Cars into gui

espi3000 committed:
Merge branch 'gui' of https://github.com/IELET1002-G2/The-Tesla-of-Zumo-Cars into gui
